pyproject.toml

```diff
@@ -86,7 +86,7 @@ ignore_missing_imports = true
 [tool.coverage.report]
```

Comment:
**commit message:**

It would be nice to have some, even brief, rationale behind this in the commit message rather than leaving it empty.

Also: 
`s/coverage fail_under value to 90/code coverage to 90%`